docs/source/installation.rst

```diff
@@ -29,14 +29,15 @@ Extra Requirements
 When installing the package via pip you can specify the following
 extra requirements:
 
-=============  =======================================================================================
-Name           Description
-=============  =======================================================================================
-``memcached``  Installs the required dependencies for the Memcached rate limiter backend.
-``rabbitmq``   Installs the required dependencies for using Dramatiq with RabbitMQ.
-``redis``      Installs the required dependencies for using Dramatiq with Redis.
-``watch``      Installs the required dependencies for the ``--watch`` flag.
-=============  =======================================================================================
+==============  =======================================================================================
+Name            Description
+==============  =======================================================================================
+``memcached``   Installs the required dependencies for the Memcached rate limiter backend.
+``rabbitmq``    Installs the required dependencies for using Dramatiq with RabbitMQ.
+``redis``       Installs the required dependencies for using Dramatiq with Redis.
+``watch``       Installs the required dependencies for the ``--watch`` flag.
+``prometheus``  Installs the required dependencies for the Prometheus statistics middleware.
```

Comment:
Prometheus can't be an optional dependency right now because it's a default middleware

Reply:
Should prometheus be made non-default, or should this change be reverted? As a user, the former would be preferable since we don't use prometheus.

Reply:
Although I understand if you'd rather make that change later.